src/bedrock/hash_ring/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+SET(ZMQ_DEPENDENCIES
```

Comment:
In the same way that we compile and point to `flproto` in `src/CMakeLists.txt`, can we compile `flzmq` or something and point to it here?

Reply:
Possibly, but maybe unnecessary. The target in this directory is the static library that everybody else links to, so it bundles zmk, proto and hash_ring into a single library. The only reason we build proto separately is because the cmake scripts for protobuf are very picky about running in the directory where you specify them, and I couldn't make it work to compile protobufs down here.